frontend/src/scenes/onboarding/OnboardingReverseProxy.tsx

```diff
@@ -0,0 +1,105 @@
+import { LemonCard, Link } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { preflightLogic } from 'scenes/PreflightCheck/preflightLogic'
+import { inviteLogic } from 'scenes/settings/organization/inviteLogic'
+
+import { InviteMembersButton } from '~/layout/navigation/TopBar/AccountPopover'
+
+import { OnboardingStepKey } from './onboardingLogic'
+import { OnboardingStep } from './OnboardingStep'
+
+const proxyDocs = [
+    {
+        title: 'AWS CloudFront',
+        link: 'https://posthog.com/docs/advanced/proxy/cloudfront',
+    },
+    {
+        title: 'Caddy',
+        link: 'https://posthog.com/docs/advanced/proxy/caddy',
+    },
+    {
+        title: 'Cloudflare',
+        link: 'https://posthog.com/docs/advanced/proxy/cloudflare',
+    },
+    {
+        title: 'Kubernetes Ingress Controller',
+        link: 'https://posthog.com/docs/advanced/proxy/kubernetes-ingress-controller',
+    },
+    {
+        title: 'Netlify',
+        link: 'https://posthog.com/docs/advanced/proxy/netlify',
+    },
+    {
+        title: 'Next.js rewrites',
+        link: 'https://posthog.com/docs/advanced/proxy/nextjs',
+    },
+    {
+        title: 'Next.js middleware',
+        link: 'https://posthog.com/docs/advanced/proxy/nextjs-middleware',
+    },
+    {
+        title: 'Vercel',
+        link: 'https://posthog.com/docs/advanced/proxy/vercel',
+    },
+    {
+        title: 'Nuxt',
+        link: 'https://posthog.com/docs/advanced/proxy/nuxt',
+    },
+]
+
+export const OnboardingReverseProxy = ({ stepKey }: { stepKey: OnboardingStepKey }): JSX.Element => {
+    const { preflight } = useValues(preflightLogic)
+    const { inviteTeamMembers } = useActions(inviteLogic)
+    const { invitesToSend, canSubmit: canSubmitInvites } = useValues(inviteLogic)
+
+    return (
+        <OnboardingStep
+            title="Reverse proxy (optional)"
+            stepKey={stepKey}
+            continueText="I've already done this"
+            showSkip
+            continueAction={() =>
+                preflight?.email_service_available &&
+                invitesToSend[0]?.target_email &&
+                canSubmitInvites &&
+                inviteTeamMembers()
```

Comment:
Since this is used identically in two places it feels like it should be something we run on step continue. So a listener in onboardingLogic

Reply:
Yep, good catch. I will look into that.